api/playbooks/environment.yml

```diff
@@ -0,0 +1,26 @@
+- name: Fetch environment information.
```

Comment:
The playbooks in this directory look like examples, maybe a reference to them needs to be included in the README or the directory could be renamed to `example-playbooks`?

Reply:
Added README for playbooks.